test/EstimateGas.sol

```diff
@@ -27,6 +27,15 @@ contract EstimateGas {
         triggerRselfdestructRefund();
     }
 
+    
+    // https://github.com/trufflesuite/ganache-cli/issues/294
+    mapping (uint => uint) public uints;
+    // Sets the uints[1] slot to a value;
+    function store(uint _uint) public { uints[1] = _uint;}
+    // Fails under testRPC provider on non empty store
+    // "callback contain no result Error: Error: base fee exceeds gas limit"
```

Comment:
Looks like these were just copied straight from https://github.com/trufflesuite/ganache-cli/issues/294, but I'm not sure if these two comments are applicable in our test, because it doesn't fail. I'd remove these for posterity's sake.